### PR TITLE
 Fixed issue #740 and #736

### PR DESCRIPTION
**Pull request by michelelacorte**
Fixed crash when placing app shortcuts and widget label on folder.

Issue #740 and #736

### CODE REVIEW

File: app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ public boolean onLongClick(View v) {
                 getContainer(), sv.getFinalInfo(),
                 new ShortcutDragPreviewProvider(sv.getIconView(), mIconShift), new DragOptions());
         dv.animateShift(-mIconShift.x, -mIconShift.y);
-
+        mLauncher.closeFolder(true);
```
Comment (divadsn) [inline app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java:113]: Fix whitespace please.

**Review by divadsn — CHANGES_REQUESTED** [review]

File: app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java (hunk 1 above)
Comment (michelelacorte) [reply app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java]: There isn't

Comment (michelelacorte) [reply app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java:113]: There isn't, and code is aligned as well

Comment (divadsn) [reply app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java:113]: image
There is indeed as GitHub shows to me.

**Review by divadsn — APPROVED** [review]

**Pull request merged**